src/dev_server.rs

```diff
@@ -11,26 +11,76 @@ use std::{
     process,
 };
 
+/// A simple HTTP server that can, optionally, watch a given command.
```

Comment:
"watch a command"? :thinking: 

```suggestion
/// A simple HTTP server useful during development.
/// It can watch the source code for changes and re-start a build command provided.
```